mobile/lib/blocs/nearby_location/nearby_location_bloc.dart

```diff
@@ -57,21 +57,19 @@ class NearbyLocationBloc
     List<AirQualityReading> airQualityReadings =
         HiveService().getAirQualityReadings();
 
-    nearByAirQualityReadings = nearByAirQualityReadings
-        .map((element) {
-          AirQualityReading referenceReading = airQualityReadings.firstWhere(
-            (reading) => reading.referenceSite == element.referenceSite,
-            orElse: () => element,
-          );
-
-          return element.copyWith(
-            pm10: referenceReading.pm10,
-            pm2_5: referenceReading.pm2_5,
-            dateTime: referenceReading.dateTime,
-          );
-        })
-        .toList()
-        .sortByDistanceToReferenceSite();
+    nearByAirQualityReadings = nearByAirQualityReadings.map((element) {
```

Comment:
In this code for nearby, could we use the endpoint I created so that we can just focus on making the endpoint better rather than have to publish a new App version? @NoahNsimbe

Reply:
Yes @Baalmart , we shall utilise the endpoint in a separate PR. Because we need to perform some QA

Reply:
Alright @NoahNsimbe